src/app.module.ts

```diff
@@ -33,6 +35,12 @@ import { RequestScopedLoggingModule } from './routes/common/logging/logging.modu
     }),
     RequestScopedLoggingModule,
   ],
+  providers: [
+    {
+      provide: APP_INTERCEPTOR,
+      useClass: RouteLoggerInterceptor,
+    },
+  ],
```

Comment:
This might look a bit cryptic but this is how we register global interceptors which require dependency injection (which is the case for the `RouteLoggerInterceptor`).

If the `RouteLoggerInterceptor` didn't depend on `ILoggingService` it could've been registered in the `AppProvider`:

```typescript
app.useGlobalInterceptors(new RouteLoggerInterceptor());
```